log.go

```diff
@@ -274,9 +308,7 @@ func (l *Logger) WithLevel(level Level) *Event {
 //
 // You must call Msg on the returned event in order to send the event.
 func (l *Logger) Log() *Event {
```

Comment:
The description if this method is no longer true.

Reply:
Sorry, I didn't understand this comment. It is still necessary to call Msg on the returned event.
Or are you referencing the comment "We use panic level..."? It's removed now.
Or something else entirely?

Reply:
Never mind, I read too fast.